### ISSUE TITLE
v2.8.0: Can't find ember-tether module after fresh install

### ISSUE THREAD

**Issue by tsteuwer** [sir-dunxalot/ember-tooltips#165]
After installing the plugin via ``ember install`` and using this code:

```
<span>
   Some text
   {{#tooltip-on-element enableLazyRendering=true}}
       <div class="some-class">
          Tooltip
       </div>
   {{/tooltip-on-element}}
</span>
```

I am getting the following error:

```
loader.js:239 Uncaught Error: Could not find module `ember-tether/components/ember-tether` imported from `ember-tooltips/components/tether-tooltip-and-popover`
```

My versions:

ember-tooltips: 2.8.0
ember-cli: 2.10.0
http_parser: 2.7.0
node: 6.0.0
v8: 5.0.71.35
uv: 1.9.0
zlib: 1.2.8
ares: 1.10.1-DEV
icu: 56.1
modules: 48
openssl: 1.0.2g
os: linux x64
3.8.6

**Comment by sir-dunxalot** [comment]
Thanks the info. Whilst we look into this, you should be able to work around this by running `ember install ember-tether` in your project.

**Comment by tsteuwer** [comment]
@sir-dunxalot  that does indeed fix the issue for now. Thank you.

**Comment by tsteuwer** [comment]
Additional information:

```
npm WARN ember-tooltips@2.8.0 requires a peer of ember-tether@0.3.1 but none was installed.
```

Is logged when doing ``npm i`` in a ember project which has it as a dependency.

**Comment by tsteuwer** [comment]
Also note, this does not allow developers to ``npm shrinkwrap`` since this package does not satisfy the missing peer dependency.

**Comment by sir-dunxalot** [comment]
Good to know, @tsteuwer. #162 is a large PR that upgrades the `ember-tether` dependency. Amongst other things, this will remove the need to register `ember-tether` as a peerDependency. Whilst #162 was not designed to fix this issue I am hopeful that it will.

**Comment by kybishop** [comment]
@sir-dunxalot just ran a fresh install with the beta; no dice on fixing the bug.

If you're interested, I'll submit a PR to automatically install ember-tether in the parent app on a fresh install via `ember install ember-tooltips`

**Comment by samdemaeyer** [comment]
@tsteuwer, @sir-dunxalot, I moved ember-tether to `dependencies` to make sure it gets installed when freshly installing the addon, don't have the dependency installed in your local project.
This PR should solve the error => #196

**Comment by sir-dunxalot** [comment]
Fixed by #196.
